packages/rocketchat-livechat/server/lib/requests.js

```diff
@@ -0,0 +1,26 @@
+import LivechatFileUpload from '../lib/LivechatFileUpload';
```

Comment:
here you can `import` the exported function (maybe makes sense to rename the file then):

`import { requestCanAccessFiles } from '../lib/LivechatFileUpload';`